app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+
+
+def copy_file(command: str) -> None:
+    names = command.split()
+    if names[1] == names[2] or\
+            len(command.split()) != 3 or\
+            command.split()[0] != "cp":
```

Comment:
You must change the order of checks:
1. Check that command `names` contains only 3 elements
2. Check that first element of `names` is "cp"
3. Check that the old name of file doesn't equal to the new name
Also, don't use command.split() a few times, please.